### PR TITLE
store, metrics: Add metrics for safetTS updating

### PR DESCRIPTION
**Pull request by Yisaer**
Signed-off-by: yisaer <disxiaofei@163.com>

What's Changed:

Add metrics for safeTS updating

Tests 

- Unit test

### Release note 

* No release note

### CODE REVIEW

File: store/tikv/kv.go [hunk 1]
```diff
@@ -414,17 +415,20 @@ func (s *KVStore) updateSafeTS(ctx context.Context) {
...
+			storeIDStr := strconv.Itoa(int(storeID))
 			if err != nil {
+				metrics.TiKVSafeTSUpdateCounter.WithLabelValues("fail", storeIDStr).Inc()
 				logutil.BgLogger().Debug("update safeTS failed", zap.Error(err), zap.Uint64("store-id", storeID))
 				return
 			}
 			safeTSResp := resp.Resp.(*kvrpcpb.StoreSafeTSResponse)
 			s.setSafeTS(storeID, safeTSResp.GetSafeTs())
+			metrics.TiKVSafeTSUpdateCounter.WithLabelValues("success", storeIDStr).Inc()
+			metrics.TiKVSafeTSUpdateStats.WithLabelValues(storeIDStr).Set(float64(safeTSResp.GetSafeTs()))
```
Comment (xhebox) [inline store/tikv/kv.go:431]: Wont timestamp exceed the float64 limit?

Comment (Yisaer) [reply store/tikv/kv.go:431]: I think it won't happen

Comment (disksing) [reply store/tikv/kv.go:431]: There should be some loss of precision.

**Review by JmPotato — APPROVED** [review]

**Review by disksing — APPROVED** [review]

**Pull request merged**